packages/react-native-sdk/README.md

```diff
@@ -115,16 +115,14 @@ Stream's video roadmap and changelog are available [here](https://github.com/Get
 
 ### 0.4 Milestones
 - [x] Expo Support
-- [ ] Leaner Chat Integration Guide
 - [ ] Regular Push Notification
 - [ ] Speaking while muted
```

Comment:
Probably remove speaking while muted also, due to - https://getstream.slack.com/archives/C04SAU8QUS0/p1695895806856849